### PR TITLE
ci: make consistent urls

### PR DESCRIPTION
**Pull request by blacha**
the $id was not pointing at where it would be published,

as part of the publishing adjust the `__STAC_VERSION__` of each repo.

### CODE REVIEW

File: extensions/template/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 - **Title:** Template
 - **Identifier:**
-  <https://stac-extensions.github.io/template/v1.0.0/schema.json>
+  <https://stac-extensions.github.io/template/__STAC_VERSION__/schema.json>
```
Comment (paulfouquet) [inline extensions/template/README.md:5]: Shound'nt be https://linz.github.io/stac/**__STAC_VERSION__/template**/schema.json ?
I mean it does not match with what is in extensions/template/examples/collection.json

File: extensions/quality/schema.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         "stac_extensions": {
           "type": "array",
           "contains": {
-            "const": "https://linz.github.io/STAC/extensions/quality/v1.0.0/schema.json"
+            "const": "https://linz.github.io/STAC/__STAC_VERSION__/quality/schema.json"
```
Comment (paulfouquet) [inline extensions/quality/schema.json:33]: Don't know if it is case sensitive ('STAC')

**Review by paulfouquet — CHANGES_REQUESTED** [review]
A couple of spelling to review

File: validate.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+node_modules/.bin/stac-node-validator \
+    --schemaMap=https://linz.github.io/stac/__STAC_VERSION__/template/schema.json=extensions/template/schema.json \
+    --schemaMap=https://linz.github.io/stac/__STAC_VERSION__/camera/schema.json=extensions/camera/schema.json \
+    --schemaMap=https://linz.github.io/stac/__STAC_VERSION__/quality/schema.json=extensions/quality/schema.json \
+    --schemaMap=https://linz.github.io/stac/__STAC_VERSION__/linz/schema.json=extensions/linz/schema.json \
+    $@
```
Comment (l0b0) [inline validate.sh:7]: This line should probably be `${@+"$@"}` to be able to optionally pass arguments with whitespace. Also, there should be a newline at EOF.

**Review by l0b0 — CHANGES_REQUESTED** [review]

**Review by l0b0 — APPROVED** [review]

**Pull request merged**